server/src/main/java/io/druid/segment/realtime/firehose/HttpFirehoseFactory.java

```diff
@@ -67,6 +80,25 @@ protected InputStream openObjectStream(URI object) throws IOException
     return object.toURL().openConnection().getInputStream();
   }
 
+  @Override
+  protected InputStream openObjectStream(URI object, long start) throws IOException
+  {
+    if (supportContentRange) {
+      final URLConnection connection = object.toURL().openConnection();
+      connection.addRequestProperty(HttpHeaders.RANGE, StringUtils.format("bytes=%d-", start));
```

Comment:
Why minus in the end? Please leave a comment.

Reply:
Added a comment.